spec/postcss/fixtures/part-nested-multiple-pseudo.pcss

```diff
@@ -0,0 +1,7 @@
+component(frame) {
+  &:hover &, &:focus & {
```

Comment:
So the `&:<pseudo> &` is some gnarly syntax we have to use in SASS because SASS is pretty rigid. Postcss is all about creating your own ideal syntax. I wonder what the ideal for this would look like and if we should shoot for it? I don't want to discount your work, you've done an amazing job. Just wondering if we can improve over the SASS syntax. Maybe something like this:

```css
component(button) {
  when(hover, focus){
    background: blue;
  }
}
```

@declandewet thoughts?